app/modules/asset_groups/resources.py

```diff
@@ -118,7 +118,17 @@ def post(self, args):
                 metadata.anonymous_submitter,
             )
 
-        asset_group.begin_ia_pipeline(metadata)
+        try:
+            asset_group.begin_ia_pipeline(metadata)
+        except HoustonException as ex:
+            asset_group.delete()
```

Comment:
This is just mock-up garbage collection, yes?

Reply:
curous about this too.

Reply:
feedback welcome on this stuff. 
This was triggered by the assets test finding more assets than expected if this test failed previously. This was cleaning up on the failure scenario to not leave things hanging around